TestTicTacToe/BoardTests.cs

```diff
@@ -0,0 +1,65 @@
+using TicTacToe;
+using Xunit;
+
+namespace TestTicTacToe
+{
+    public class BoardTests
+    {
+        [Fact]
+        public void Should_Place_Marker_In_The_Correct_Position_If_Empty_Cell()
+        {
+            // Arrange 
+            var board = new Board();
+            var move = new Move(1 - Constants.IndexingAdjustment, 1 - Constants.IndexingAdjustment);
+            
+            var expectedBoard = new[,]
+            {
+                {1, 0, 0},
+                {0, 0, 0},
+                {0, 0, 0}
+            };
+        
+            // Act
+            board.PlaceMarker(move, 1);
```

Comment:
using `int` to represent a player is fine, but I think there could be better ways to `reveal intent`

Reply:
eg. just by reading this function the only way I found out `1` probably means player 1 is by looking at the definition of `PlaceMarker`

Reply:
Agree with Alvin, you have to look at several of the tests to go "Oh, 1 is probably player one and 2 is probably player two"